src/tid-ext.adoc

```diff
@@ -0,0 +1,104 @@
+== "Zstid Extension for Thread Identification
+
+{tid_ext_name} is an optional extension to the RISC-V base ISA.
+Implementations that support {cheri_base_ext_name} and {tid_ext_name}.
+define a variant of the CHERI ISA that allows for software compartmentalization
+of CHERI programs.
+
+=== Control and Status Registers (CSRs)
+
+{tid_ext_name} adds two new CSRs to implement a trusted thread
+identifier (TID) used in compartmentalization. These CSRs are listed in
+xref:tid-scsrnames-added[xrefstyle=short] and
+xref:tid-ucsrnames-added[xrefstyle=short].
+
+[[tid-scsrnames-added]]
+.Added supervisor-mode CSRs in {tid_ext_name}
+[%autowidth,float="center",align="center",cols="<,<,<,<,<",options="header"]
+|===
+|{tid_ext_name} CSR|Address|Prerequisites|Permissions|Description
+|<<stid>>|0x580|S-mode|SRW, <<asr_perm>>|Supervisor Thread Identifier
+|===
+
+[[tid-ucsrnames-added]]
+.Added user-mode CSRs in {tid_ext_name}
+[%autowidth,float="center",align="center",cols="<,<,<,<,<",options="header"]
+|===
+|{tid_ext_name} CSR|Address|Prerequisites|Permissions|Description
+|<<utid>>|0xC80|U-mode|URO|User Thread Identifier
+|===
+
+=== Supervisor-Level CSRs
+
+[#stid,reftext="stid"]
+==== Supervisor Thread Identifier (stid)
+
+The <<stid>> register is an SXLEN-bit read-write register. It is used to
+identify the current thread. The reset value of this register is
+UNSPECIFIED.
+
+.Supervisor thread identifier register
+include::img/stidreg.edn[]
+
+[#stidc,reftext="stidc"]
+==== Supervisor Thread Identifier Capability (stidc)
```

Comment:
I'd move the capability versions below the integer ones and add a sentence above along the lines of 

`When {cheri_base_ext} is implemented, the {tid_ext_name} CSRs are extended as follows:`